tests/test_main.py

```
@@ -1885,12 +2579,12 @@ def test_main_signal_handler_sets_shutdown_flag(self):
         real_get_running_loop = asyncio.get_running_loop
 
         def _patched_get_running_loop():
-            loop = real_get_running_loop()
+            loop: Any = real_get_running_loop()
             if not hasattr(loop, "_signal_handler_patched"):
 
-                def _fake_add_signal_handler(_sig, handler):
-                    captured_handlers.append(handler)
-                    handler()
+                def _fake_add_signal_handler(sig, callback, *args):
```

Ruff (0.14.14)

Unused function argument: `sig`

(ARG001)